docs/guide/existing-game.md

```diff
@@ -1,11 +1,20 @@
 **This page is under construction!**
 
-## Summary
-* Tools to port existing games are in progress!
-    * [rbxlx-to-rojo](https://github.com/rojo-rbx/rbxlx-to-rojo)
-    * `rojo export` ([issue #208](https://github.com/rojo-rbx/rojo/issues/208))
-* Can port as much or as little of your game as you like
-    * Rojo can manage just a slice of your game!
-* Some Roblox idioms aren't very well supported
-    * Redundant copies of scripts don't work well with files
-    * Having only a couple places with scripts simplifies your project dramatically!
+# Using rbxlx-to-rojo
+## Setup
+Before you can use rbxlx-to-rojo, you need to have Rojo 0.5.0 Alpha 12 to use the tool.
+You also need a rbxlx place file that has a script. If there aren't any scripts in the rbxlx file, rbxlx-to-rojo will return an error.
+
+Download the latest release of rbxlx-to-rojo here: https://github.com/rojo-rbx/rbxlx-to-rojo/releases
+## Porting the game
+Let's assume we have a rbxlx file named "game", with a script in ServerScriptService.
+
+### Steps to port the game:
+1. Double-click on rbxlx-to-rojo on wherever you installed it.
+2. Select a place file.
+3. Select the path where you want to place it.
+
+If you followed the steps correctly, you should see something that looks like this:
+https://cdn.discordapp.com/attachments/609230525604954135/610155628786221117/unknown.png
```

Comment:
This should be an actual image, not just a link (`![]()`)

Reply:
Alright, I'll fix it.

Reply:
Fixed. Check the latest commit.